crates/consensus/Cargo.toml

```diff
@@ -11,14 +11,19 @@ readme = "README.md"
 reth-primitives = { path = "../primitives" }
 reth-interfaces = { path = "../interfaces" }
 reth-provider = { path = "../storage/provider" }
-reth-rlp = {path = "../common/rlp"}
+reth-rlp = { path = "../common/rlp" }
+reth-rpc-types = { path = "../net/rpc-types" }
```

Comment:
I wonder if we should keep the EngineApi separately, but perhaps not worth it now.